src/System.Linq/tests/ToListTests.cs

```diff
@@ -25,6 +25,16 @@ public void ToList_AlwaysCreateACopy()
         }
 
 
+        [Fact]
```

Comment:
This will have to be excluded from netfx (and possibly other cases) as these tests are run against that, but it won't have this addition.